core/src/main/scala/tofu/lift/IsoK.scala

```diff
@@ -30,4 +30,11 @@ object IsoK {
     def to[A](fa: F[A]): F[A]   = fa
     def from[A](ga: F[A]): F[A] = ga
   }
+
+  def byFunK[F[_], G[_]](fg: F ~> G)(gf: G ~> F): IsoK[F, G] =
+    new IsoK[F, G] {
+      override def to[A](fa: F[A]): G[A] = fg(fa)
+
+      override def from[A](ga: G[A]): F[A] = gf(ga)
+    }
```

Comment:
Let's apply consistent formatting to similar code pieces. I mean use of `override` and blank lines.